src/index/base.cpp

```diff
@@ -23,7 +23,7 @@ static void FatalError(const char* fmt, const Args&... args)
     std::string strMessage = tfm::format(fmt, args...);
     SetMiscWarning(Untranslated(strMessage));
     LogPrintf("*** %s\n", strMessage);
-    AbortError(_("A fatal internal error occurred, see debug.log for details"));
+    AbortError(_(strMessage.c_str()));
```

Comment:
A string can't be translated. Only a string literal.

Reply:
So the options are:
* Mark this as `Untranslated()`
* Pass in the translated string from the caller
* Something else?

Reply:
I opted to mark it as `Untranslated`.